lib/index.ts

```diff
@@ -203,6 +225,12 @@ export class Front {
 		return httpServer;
 	}
 
+	// Utility method for occasions where we have the actual url, eg `_links`
+	public get(url: string, callback?: Callback<Object>): Promise<Object> {
```

Comment:
Probably worth calling this `getUrl` or `getLinks` as `get` is a bit ambiguous.

Reply:
ack, will do